test/e2e/storage/csi_mock_volume.go

```diff
@@ -1360,9 +1360,11 @@ func parseMockLogs(cs clientset.Interface, namespace, driverPodName, driverConta
 	logLines := strings.Split(log, "\n")
 	var calls []mockCSICall
 	for _, line := range logLines {
-		if !strings.HasPrefix(line, "gRPCCall:") {
+		index := strings.Index(line, "gRPCCall:")
+		if index == -1 {
 			continue
 		}
+		line = line[index:len(line)]
```

Comment:
At this point you already know that `line[index:len(line)]` starts with `gRPCCall:`, therefore you can combine this and the next line.

```
const grpcCall = "gRPCCall:"

    index := strings.Index(line, grpcCall)
    ...
    line = line[index+len(grpcCall):len(line)]
```

Reply:
Done.